<launch>
  <include file="$(find jsk_pr2_startup)/jsk_pr2_rocon/pr2s_rocon_common/pr1040_rocon.launch">
    <arg name="config_path" value="$(find tableclothdemo)/config/tableclothdemo_pr1040_config.yaml" />
  </include>
  <!-- rocon communication -->
  <node pkg="topic_tools" type="relay" name="rename_bb"
        args="/bb_repub /raw_tablecloth_bb" />
  <!-- for searching tablecloth -->
  <node pkg="tableclothdemo" name="raw_boundingbox_to_margined_boundingbox"
        type="raw_boundingbox_to_margined_boundingbox.py" />
  <node pkg="tableclothdemo" name="boundingbox_to_transform_stamped"
        type="boundingbox_to_transform_stamped.py" />
  <node pkg="tableclothdemo" name="search_four_corners" type="search-four-corners.py" />
</launch>
